src/config/userTypes.ts

```diff
@@ -38,7 +38,7 @@ export interface IHacker {
       degree: string;
       fieldOfStudy: string;
       graduationYear: number;
-      jobInterest: JobInterest;
+      jobInterest: string;
```

Comment:
Why was this switched to string?

Reply:
in case we make it creatable in the future I guess and just to match the other properties